### PR TITLE
feat: Pbi 19 improve api monitor stats frontend

### PR DESCRIPTION
**Pull request by ferdifdi**
### Describe your changes
implement if there is no monitor
implement yellow bar in our logic

image
image

### Backlog name
PBI-9-improve_API_monitor_stats-frontend

### Checklist
- [X] I have performed a self-review of my code
- [X] Code successfully run on local
- [X] Feature / changes tested on local
- [X] No failing unit test
- [ ] Passed UAT Test
- [x] Sonarqube tested 
- [ ] Required any changes to environment variable

### CODE REVIEW

**Review by luckysusanto — APPROVED** [review]
LGTM

**Review by andrew-susanto — APPROVED** [review]

File: src/components/success_rate/index.js [hunk 1]
```diff
@@ -10,14 +10,22 @@ const SuccessRate = ({success,failed})=> {
...
                         <div class={style['green-bar']}>
                             
                         </div>
                         :
-                        <div class={style['red-bar']}>
+                        <>
+                            {(success == 0)?
+                                <div class={style['red-bar']}>
+                                    
+                                </div>
```
Comment (andrew-susanto) [inline src/components/success_rate/index.js]:
```suggestion
                                <div class={style['red-bar']} />
```
ini coba diimprove kayak gini kayaknya bisa fer

File: src/components/success_rate/index.js [hunk 2]
```diff
@@ -10,14 +10,22 @@ const SuccessRate = ({success,failed})=> {
                 </div>
                 :
                 <>
-                    {(success >= failed)?
+                    {(failed == 0)?
                         <div class={style['green-bar']}>
                             
                         </div>
```
Comment (andrew-susanto) [inline src/components/success_rate/index.js]:
```suggestion
                        <div class={style['green-bar']} />
```

**Review by luthfifahlevi — APPROVED** [review]
Nice

**Review by HugoIr — APPROVED** [review]
good

**Pull request merged**